diff --git a/node_modules/@mui/material/Button/Button.d.ts b/node_modules/@mui/material/Button/Button.d.ts
index 63b0789..9a9ae19 100644
--- a/node_modules/@mui/material/Button/Button.d.ts
+++ b/node_modules/@mui/material/Button/Button.d.ts
@@ -28,7 +28,7 @@ export interface ButtonOwnProps {
    * @default 'primary'
    */
   color?: OverridableStringUnion<
-    'inherit' | 'primary' | 'secondary' | 'success' | 'error' | 'info' | 'warning',
+    'inherit' | 'primary' | 'secondary' | 'success' | 'error' | 'info' | 'warning' | string,
     ButtonPropsColorOverrides
   >;
   /**
diff --git a/node_modules/@mui/material/Select/Select.d.ts b/node_modules/@mui/material/Select/Select.d.ts
index c39944e..5825643 100644
--- a/node_modules/@mui/material/Select/Select.d.ts
+++ b/node_modules/@mui/material/Select/Select.d.ts
@@ -143,7 +143,7 @@ export interface BaseSelectProps<Value = unknown>
    * If the value is an object it must have reference equality with the option in order to be selected.
    * If the value is not an object, the string representation must match with the string representation of the option in order to be selected.
    */
-  value?: Value | '';
+  value?: Value | '' | number;
   /**
    * The variant to use.
    * @default 'outlined'
@@ -193,7 +193,7 @@ export type SelectProps<Value = unknown> =
  * - [Select API](https://mui.com/material-ui/api/select/)
  * - inherits [OutlinedInput API](https://mui.com/material-ui/api/outlined-input/)
  */
-export default function Select<Value = unknown>(
+export default function Select<Value = unknown|number>(
   props: SelectProps<Value>,
 ): JSX.Element & {
   muiName: string;
